src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -79,8 +91,9 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP)
     {
       if (type != "String")
       {
-        LM_W(("Bad Input (bad type for EntityId::id"));
-        eP->errorCode.fill("JSON Parse Error IV", "bad type for EntityId::id");
+        LM_W(("Bad Input (invalid JSON type for EntityId::id"));
+        eP->errorCode.fill("JSON Parse Error", "invalid JSON type for EntityId::id");
```

Comment:
I think that V2 is not using the "EntityID::id" format, in favour of a lean syntax... just "invalid JSON type for entity id" would be ok.